content/en/docs/ops/troubleshooting/istioctl-describe/index.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Understand your Mesh with istioctl describe
+description: Shows you how to use istioctl describe to verify the configurations of a pod in your mesh.
+weight: 90
+keywords: [traffic-management, istioctl, debugging, kubernetes]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+In Istio 1.3, we included the [`istioctl experimental describe`](/docs/reference/commands/istioctl/#istioctl-experimental-describe-pod)
+command. This CLI command provides you with the information needed to understand
+the configuration impacting a {{< gloss >}}pod{{< /gloss >}}. This guide shows
+you how to use this experimental sub-command to see if a pod is in the mesh and
+verify its configuration.
+
+The basic usage of the command is as follows:
+
+{{< text bash >}}
+$ istioctl experimental describe <pod-name>[.<namespace>]
+{{< /text >}}
+
+You can replace `experimental` with `x` for convenience and you can use the `istioctl`
+`-n` option to specify a non-default namespace as an alternative to appending the namespace
+to the pod name.
```

Comment:
Consider losing this paragraph. All of the istioctl commands have these features.